configs/operations/github-activity-events-monitor.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Activity Events Monitor
+
+events:
+  - issues.opened
+  - issues.closed
+  - issues.labeled
+  - issues.unlabeled
+  - issues.transferred
+  - issues.assigned
+  - issue_comment.created
+  - push
+  - pull_request.closed
+  - pull_request.opened
+  - pull_request.labeled
+  - pull_request.unlabeled
+  - pull_request.assigned
+  - pull_request_review.submitted
+  - pull_request_review_comment.created
+  - gollum
+  - release.released
+  - project.edited
```

Comment:
These needs to be enabled in opensearch-infra app before take effect.

Reply:
Right, it would be great if you could enable these. If we don't want to enable it's ok because most of the activity is coming from issue and pull_request

Reply:
We probably can ignore the `project.edited` for now and `release.released`.

I would start with the `issues`, `pull_request` and `gollum` now and back for a few days.

You can comment out the others.